content/go/concepts/interfaces/interfaces.md

```diff
@@ -0,0 +1,264 @@
+---
+Title: 'Interfaces'
+Description: 'An interface is a set of method signatures.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Interfaces'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+An interface is a set of method signatures.
+
+```go
```

Comment:
Can we create an h2 here that is along the lines of "Creating an Interface" and nest the relevant code and text underneath